data/json/items/resources/metal.json

```diff
@@ -166,8 +166,8 @@
     "name": "magnesium powder",
     "name_plural": "magnesium powder",
     "description": "A gray powder composed of the highly flammable alkaline earth metal magnesium.  Useful for constructing flares and similar items.",
-    "weight": 4,
-    "//": "Density 1.74g/cm³ ~ 0.44kg/250ml @ stack 100 = 4g/unit",
+    "weight": 1.1,
```

Comment:
Is it even working if you say above that 1.75 is rounded down to 1?

Reply:
No it doesn't work.
But it doesn't affect anything ( the other incremental value was there for a long time I think ). And if something changes and that 0.1 will be taken into account then it will be perfect + I didn't want to continue the "//" comment it already is ridiculously long and convoluted. ( I am trying to come up with a better "//" comment to explain the logic, I can take suggestions )
That being said, if its a problem or something I have nothing against removing the 0.1 .

Reply:
I see. Thanks.